README.md

```diff
@@ -1 +1,44 @@
-# viam-csi
\ No newline at end of file
+# viam-csi
+> viam module for csi cams
+
+![](./etc/viam-server.png)
+
+
+> **Warning** Currently in ALPHA and only supports Jetson. Please open an issue if you run into problems.
```

Comment:
Ok sounds good.

I added a warning since the underlying viam-cpp-sdk is still in alpha.

https://github.com/viamrobotics/viam-cpp-sdk#instability-notice

Reply:
Ah, sounds good